src/azure-account.ts

```diff
@@ -376,9 +380,21 @@ export class AzureLoginHelper {
 		if (selected) {
 			const config = workspace.getConfiguration('azure');
 			if (config.get('cloud') !== selected.environment.name) {
+				if (selected.environment.name === azureCustomCloud) {
+					const armUrl = await window.showInputBox({
+						prompt: localize('azure-account.enterArmUrl', "Enter the Azure Resource Manager endpoint"),
```

Comment:
I suggest adding `ignoreFocusOut: true` here as well, to avoid losing the prompt easily while the user finds the url they need